aten/src/ATen/core/jit_type.h

```diff
@@ -1130,18 +1130,19 @@ struct CAFFE2_API TupleType : public NamedType {
 
 struct EnumType;
 using EnumTypePtr = std::shared_ptr<EnumType>;
+using EnumNameValue = std::pair<std::string, IValue>;
```

Comment:
Also, I was wondering if we can use `const std::string` and `const IValue` to help enforce immutability. Or maybe `enumNamesValues` should return a `const ArrayRef`?

Reply:
Good point! Both done.